docs/testplan/PMON-Services-Daemons-test-plan.md

```diff
@@ -136,8 +136,8 @@ If a daemon doesn't support the version command, then the test assums the versio
 
 ## Test case # 1 All expected daemons are running
 ### Test objective
-Verify each daemon status in PMON docker using "check_pmon_daemon_status()" in "tests/common/platform/daemon_utils.py"
-Extend "check_pmon_daemon_status()" if there is any gap to follow the steps below.
+Verify all daemon staus in PMON container using "check_pmon_daemon_status()" in "tests/common/platform/daemon_utils.py"
```

Comment:
s/staus/status

Suggest rewording to "Verify the status of all daemons in PMon container using ..."